cassandra-dac/src/main/java/org/sunbird/cassandra/CassandraOperation.java

```diff
@@ -111,6 +121,19 @@ Response getRecordsByProperty(
       List<Object> propertyValueList,
       List<String> fields);
 
+  /**
+   * Fetch records with specified columns (select all if null) for given column name with matching
```

Comment:
Change comment:
Fetch records for given indexed column name and value.
...
propertyValue Value to be used for matching in select query